roles/wildfly_app_deploy/tasks/standalone.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Check if application is already deployed (Standalone)
+  block:
+    - name: Query deployment status
+      ansible.builtin.include_role:
+        name: wildfly_utils
+        tasks_from: jboss_cli.yml
+      vars:
+        jboss_cli_query: "'/deployment={{ wildfly_app_deploy_name }}:read-resource'"
+        jboss_cli_timeout: 10
+        jboss_cli_changed_when: false
+      register: deployment_check
+  rescue:
+    - name: Deployment does not exist
+      ansible.builtin.set_fact:
+        deployment_exists: false
+  always:
+    - name: Set deployment exists flag based on check result
+      ansible.builtin.set_fact:
+        deployment_exists: "{{ deployment_check.failed | default(true) == false }}"
+      when: deployment_exists is not defined
+
+- name: Deploy Application (Standalone)
+  ansible.builtin.include_role:
+    name: wildfly_utils
+    tasks_from: jboss_cli.yml
+  vars:
+    jboss_cli_query: "'deploy /tmp/{{ wildfly_app_deploy_name }} --force'"
+    jboss_cli_timeout: 30
```

Comment:
Is this the timeout between Ansible and JBoss CLI or the wait for the deployment to finish? If the latter, this should be higher. It's worth checking what is the default.